pkg/coremodel/dashboard/coremodel.cue

```diff
@@ -192,6 +192,58 @@ seqs: [
 					steps: [...#Threshold] @reviewme()
 				} @cuetsy(kind="interface") @reviewme()
 
+				// TODO docs
```

Comment:
Note that all changes in this file should be functional no-ops. All the refactoring here is basically just moving declarations around so that codegen produces all the named types that currently exist in handwritten form.